src/validation/userValidation.js

```diff
@@ -0,0 +1,9 @@
+import { check } from 'express-validator';
+
+const loginValidation = [
+  check('email').isEmail().trim().withMessage('input a valid email address'),
```

Comment:
Please update the first letter of each message to capital letter

Reply:
ok thanks